test/shared-method.test.js

```diff
@@ -79,6 +79,30 @@ describe('SharedMethod', function() {
   });
 
   describe('sharedMethod.invoke', function() {
+    it('returns default value on custom remote method when parameter is not passed', function(done){
+      var defaultValue = 100;
+      var method = givenSharedMethod(
+        function(num, next) { 
+          return new Promise(function(resolve, reject) {
```

Comment:
I think this can test be simplified a bit to something line the following:

```js
function(num) {
  if (typeof num !== 'number') {
    return Promise.reject(new Error(...));
  }
  return Promise.resolve({ num: num });
}
```